### PR TITLE
[CC-21880] Update cart totals based on shipment selector

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
documentation for the epic: https://spryker.atlassian.net/browse/CC-21880
Ticket: https://spryker.atlassian.net/browse/CC-24078

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/cart-and-checkout/manage-using-glue-api/check-out/submit-checkout-data.md [hunk 1]
```diff
@@ -257,6 +261,84 @@ To retrieve all available shipment methods, submit checkout data with one or mor
 ```
 </details>
 
+<details>
+<summary markdown='span'>Request sample: submit checkout data with the guest customer's cart data.
```
Comment (VadymSachenko) [inline docs/pbc/all/cart-and-checkout/manage-using-glue-api/check-out/submit-checkout-data.md:265]:
```suggestion
<summary markdown='span'>Request sample: submit checkout data with the guest customer's cart data.</summary>
```

[the remark this answers is not in the document]
Comment (VadymSachenko) [reply docs/pbc/all/cart-and-checkout/manage-using-glue-api/check-out/submit-checkout-data.md:265]: @sakharova-yuliia FIxed

File: docs/pbc/all/cart-and-checkout/manage-using-glue-api/check-out/submit-checkout-data.md [hunk 2]
```diff
@@ -958,6 +1038,154 @@ In case of a successful update, the endpoint responds with information that can
...
+{
+    "data": {
+        "type": "checkout-data",
+        "id": null,
+        "attributes": {
+            "addresses": [],
+            "paymentProviders": [],
+            "shipmentMethods": [],
+            "selectedShipmentMethods": [],
+            "selectedPaymentMethods": []
```
[the remark this answers is not in the document]
Comment (VadymSachenko) [reply docs/pbc/all/cart-and-checkout/manage-using-glue-api/check-out/submit-checkout-data.md:1054]: @sakharova-yuliia I believe we should add the missing data. Can you provide it or just add it here yourself?

[the remark this answers is not in the document]
Comment (sakharova-yuliia) [reply docs/pbc/all/cart-and-checkout/manage-using-glue-api/check-out/submit-checkout-data.md]: Adjusted responses and requests examples.

**Review by sakharova-yuliia — APPROVED** [review]

**Review by lenadoc — APPROVED** [review]

**Pull request merged**